commands/post-install.ts

```diff
@@ -32,6 +33,12 @@ export class PostInstallCommand implements ICommand {
 
 			this.checkSevenZip().wait();
 			this.$commandsService.tryExecuteCommand("autocomplete", []).wait();
+
+			var sysInfo = this.$sysInfo.getSysInfo();
+			if (!sysInfo.adbVer) { console.warn("Cannot find adb in the path. The built-in one will be used, which may be incompatible with the latest Android SDK or Genymotion"); }
+			if (!sysInfo.antVer) { console.warn("Cannot find ant. You cannot build Android projects."); }
+			if (!sysInfo.javaVer) { console.warn("Cannot find java. You cannot build Android projects."); }
+			if (hostInfo.isDarwin() && !sysInfo.xcodeVer) { console.warn("Cannot find Xcode. You cannot build iOS projects and use the iOS Simulator."); }
```

Comment:
Can we tweak the messages a little bit to provide next steps for action? For example: `You can download the latest version of Xcode from the App Store`.